src/OrchardCore.Modules/OrchardCore.Resources/ResourceManagementOptionsConfiguration.cs

```diff
@@ -1042,8 +1042,8 @@ private ResourceManifest BuildManifest()
                 "https://cdn.jsdelivr.net/npm/vue-multiselect@3.2.0/dist/vue-multiselect.umd.js"
             )
             .SetCdnIntegrity(
-                "sha256-WPNu0JAREY1Irusgtihcuzati2fLtA70vyK1TDs55PY=",
-                "sha256-bgP1jhSlmX0vEY01VGFi6FMuUzU5Vbd1AdXKm9o0i4Q="
+                "sha384-7HofOP/3o+/jDuC4RmU64mOramSz9btGjs7S9tsRd4U6lerpHI0P1jEFjjvxIBi6",
```

Comment:
The only comment that I have is why you changed these SHAs?!!

Reply:
Alright!